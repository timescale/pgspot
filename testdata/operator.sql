
CREATE OPERATOR #? (LEFTARG=bool,RIGHTARG=bool,FUNCTION=boolge);

