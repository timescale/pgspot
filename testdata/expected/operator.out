PS017: Unqualified object reference: #? at line 2

Errors: 0 Warnings: 1 Unknown: 0
